tests/functional/testMultihop.js

```diff
@@ -63,12 +68,6 @@ describe('Server list', function() {
             queries.screenHome.serverListView.VPN_COLLAPSIBLE_CARD,
             'expanded') === 'false');
 
-    await vpn.waitForQuery(
-        queries.screenHome.serverListView.ENTRY_BUTTON.visible());
-
-    await vpn.waitForQuery(
-        queries.screenHome.serverListView.EXIT_BUTTON.visible());
-
```

Comment:
these checks didn't seem to serve any purpose, so I removed them and renamed this test